pod-e2e.sh

```diff
@@ -10,7 +10,7 @@ POD_NAME="pod_e2e"
 
 echo "cleaning up and recreating pod"
 podman pod rm -f ${POD_NAME}
-podman pod create --name=${POD_NAME} --publish 8000
+podman pod create --name=${POD_NAME} --publish 8000:8000
 
 POD_NAME="pod_e2e" ./backend/tests/opensearch_ocp.sh
 podman run -d --pod=${POD_NAME} --name=back e2e-backend
```

Comment:
And, again, the only thing "functional/e2e test-y" about the backend container is that we map a specific `ocpperf.toml` when we deploy it. There's no need for a separate container image name or build.

```bash
podman run -d ${POD} --name="${POD_NAME}-back" -v "${CPT_CONFIG}:/backend/ocpperf.toml:Z" localhost/backend
```

Reply:
This is related to what I was asking the other day about the `functional.containerfile`. I can't override `ENTRYPOINT ["pytest", "-s", "tests/functional"]` to execute my script to seed the database. I wasn't sure how to achieve both its original behavior for pytest and my desired behavior of database seeding.

Reply:
I believe I figured out how to resolve the issues with using the `functional.containerfile`.